node/flatpak_node_generator/providers/__init__.py

```diff
@@ -33,6 +34,14 @@ def parse_git_source(self, version: str, from_: Optional[str] = None) -> GitSour
         if not new_url.netloc:
             path = new_url.path.split('/')
             new_url = new_url._replace(netloc=path[0], path='/'.join(path[1:]))
+        # Replace https://git@github.com:ianstormtaylor/to-camel-case.git
+        # with    https://git@github.com/ianstormtaylor/to-camel-case.git
+        # for git+ssh URLs
+        if ':' in new_url.netloc:
+            netloc_split = new_url.netloc.split(':')
```

Comment:
this will break if there are extra colons (I mean, there really *shouldn't* be, but alas)
```suggestion
            netloc_split = new_url.netloc.split(':', 1)
```

(note that the branch immediately above also handles this, but...it does so kinda poorly, by comparison)